uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/viewmodel/RealFeedViewModel.kt

```diff
@@ -135,18 +102,13 @@ class RealFeedViewModel @Inject constructor(
                 is FeedViewModel.Event.ChangePlayingPodcastState -> {
                     val state = playingPodcastState.value
                     val isPlaying = event.feedItem.id == state?.id && state.isPlaying
-
                     val newState = PlayingPodcastState(
                         id = event.feedItem.id,
                         url = event.feedItem.podcastLink(),
                         isPlaying = isPlaying.not()
                     )
-                    if (newState.isPlaying) {
-                        playPodcast(newState.url, state?.url == newState.url)
-                    } else {
-                        pausePodcast()
-                    }
                     playingPodcastState.value = newState
+                    effectChannel.send(FeedViewModel.Effect.ControlFmPlayer(newState))
```

Comment:
Thank you for separating!  👍 
For example, if you want to play in the background, it will be difficult if the feedViewModel manages the playback state. 👀 
Therefore, is it possible to move the playback status as well?

Reply:
indeed! That state management method is a better way 👍 
I will fix it now. Thank you for the advice!